bin/compile-extension-mongodb

```diff
@@ -6,8 +6,8 @@ source $(dirname $0)/compile-extensions-common
 travis_time_start
 
 if [[ $VERSION =~ ^7 || $VERSION =~ ^master$ ]]; then
-	git clone https://github.com/mongodb-labs/mongo-php-driver-prototype.git
-	pushd mongo-php-driver-prototype
+	git clone https://github.com/mongodb/mongo-php-driver.git
+	pushd mongo-php-driver
```

Comment:
Similar to my question in https://github.com/travis-ci/php-src-builder/pull/4/files#r138070257, is there any reason we can't simply `pecl_install mongodb` for all PHP versions?

This looks as if it's pulling the `master` branch for all PHP 7.x builds.

PHP 7.x support has been available since [1.1.1](https://github.com/mongodb/mongo-php-driver/releases/tag/1.1.1), so we should able to rely on PECL to pull in a recent stable version.

Reply:
PHP 7 support was not great when we started building these, so building from source was a way to make it happen at all.

We have not kept up to date since.

Reply:
I think this PR is still valid, as "mongodb/mongo-php-driver" has been the canonical repository since we renamed the old extension to "mongodb/mongo-php-driver-legacy" many moons ago.

I can follow up with a later PR that opts to use `pecl_install mongodb` for most cases and only falls back to building the extensions' master branch from source if the PHP version is also master (I assume that is a nightly build).

[`bin/compile-extension-mongo`](https://github.com/travis-ci/php-src-builder/blob/default/bin/compile-extension-mongo) is probably fine as-is (checking out the 1.6.16 tag and building from source). I don't expect we'll have another release for the legacy driver (that was the first in over a year).